cfpb/design-system
# Remove govuk tabs dependency and roll our own

## Changes

- Remove govuk tabs dependency and create our own Tabs component. 

## Testing

1. PR checks should pass.
2. Check PR preview and see that the tabs under the "SHOW DETAILS" toggle work.
3. Edit a page and see that the tabs work in the admin preview.

## Todo

- The tabs CSS should be tweaked to fix https://github.com/cfpb/design-system/issues/1276
